.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Release CI
+on:
+  push:
+    branches:
+      - master
+      - next
+      - beta
+      - alpha
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Install dependencies
+        run: npm ci
+      - name: Lint
+        run: npm run lint
+      - name: Test
+        run: npm run test
+      - name: Release
+        uses: cycjimmy/semantic-release-action@v2
+        with:
+          semantic_version: 16
+        env:
+          GITHUB_TOKEN: ${{ secrets.SEMANTIC_RELEASE_GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.SEMANTIC_RELEASE_NPM_TOKEN }}
```

Comment:
Did the secrets already exist there or did you have to migrate them?

Reply:
I didn't do anything special there and am not familiar with how the secrets work... should I have had to do something?

Reply:
I checked; there do appear to be pre-existing Github secrets by those names, so it _should_ work :crossed_fingers: